mini-cap/src/pages/CondoDetails.jsx

```diff
@@ -220,6 +251,28 @@ export default function CondoDetails(){
 										<button className="delete-button" data-testid="delete-button-test" onClick={() => handleClickDelete()}>Delete</button>
 									</>)}
 							</div>
+							<div>
+							{role !== MANAGEMENT_COMPANY && (
```

Comment:
change this so that both renter/owners and management companies can see the financial details